src/components/advanced-search/advanced-search-form.vue

```diff
@@ -0,0 +1,200 @@
+      <v-col>
+        <span class="title-bold-16">Requested Business Name</span>
+        <v-text-field
+          filled
+          class="mt-2"
+          label="Requested Business Name"
+          v-model="businessName"
+        />
+      </v-col>
+    </v-row>
+
+    <v-row no-gutters class="mt-n4">
+      <v-col>
+        <span class="title-bold-16">Applicant Name</span>
+        <v-row>
+          <v-col>
+            <v-text-field
+              filled
+              label="First Name"
+              v-model="applicantFirstName"
+            />
+          </v-col>
+          <v-col>
+            <v-text-field
+              filled
+              label="Last Name"
+              v-model="applicantLastName"
+              :rules="applicantLastNameRules"
+            />
+          </v-col>
+        </v-row>
+      </v-col>
+    </v-row>
+
+    <v-row no-gutters class="mt-n4">
+      <v-col>
+        <span class="title-bold-16 mr-4">Submitted Date Range</span>
+        <span v-if="hasAppliedDates" id="clear-date-btn" class="text-body-3 app-blue" @click="clearDates()">
+          Clear <v-icon small>mdi-close</v-icon>
+        </span>
+        <v-row>
+          <v-col>
+            <v-text-field
+              filled
+              label="Start Date"
+              hint="YYYY-MM-DD"
+              persistent-hint
+              append-icon="mdi-calendar"
+              v-model="startDate"
+              @click="dateDialog = true"
+              @click:append="dateDialog = true"
+              v-on:keydown="$event.preventDefault()"
+            />
+          </v-col>
+          <v-col>
+            <v-text-field
+              filled
+              label="End Date"
+              hint="YYYY-MM-DD"
+              persistent-hint
+              append-icon="mdi-calendar"
+              v-model="endDate"
+              @click="dateDialog = true"
+              @click:append="dateDialog = true"
+              v-on:keydown="$event.preventDefault()"
+            />
+          </v-col>
+        </v-row>
+      </v-col>
+    </v-row>
+
+  </v-form>
+</template>
+
+<script lang='ts'>
+import { Component, Emit, Prop, Vue, Watch } from 'vue-property-decorator'
+import AdvancedSearchDates from '@/components/advanced-search/advanced-search-dates.vue'
+
+// Interfaces
+import { AdvancedSearchI, FormType, StartEndDatesI } from '@/interfaces'
+
+@Component({
+  components: {
+    AdvancedSearchDates
+  }
+})
+export default class AdvancedSearchForm extends Vue {
+  // Refs
+  $refs!: {
+    advancedSearchForm: FormType
+  }
+
+  /** Prompt the form to pass up the form data. */
+  @Prop() readonly promptSubmit: boolean
+
+  // Local Properties
+  private isValid = false
+  private dateDialog = false
+
+  // Local model
+  private businessName = ''
+  private applicantFirstName = ''
+  private applicantLastName = ''
+  private startDate = ''
+  private endDate = ''
+
+  /** Validation rules for Applicant last name. */
+  private applicantLastNameRules = []
```

Comment:
Will this be used?

Reply:
Yes, they are applied ONLY if the client enters a first name, line 140.

Reply:
Gotcha. I missed that the first time. You are correct :)